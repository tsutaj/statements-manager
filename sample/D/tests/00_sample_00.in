3
3 9
100 101
1 10
