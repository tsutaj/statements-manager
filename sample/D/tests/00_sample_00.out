12
201
11
